src/entity/projectile/SplashPotion.php

```diff
@@ -98,8 +98,8 @@ protected function onHit(ProjectileHitEvent $event) : void{
 
 		if($hasEffects){
 			if(!$this->willLinger()){
-				foreach($this->getWorld()->getNearbyEntities($this->boundingBox->expandedCopy(4.125, 2.125, 4.125), $this) as $entity){
-					if($entity instanceof Living && $entity->isAlive()){
```

Comment:
I don't see anywhere else this is checked.

Reply:
`World::getCollidingEntities()` calls `Entity::canBeCollidedWith()` this checks that the entity is alive:
https://github.com/pmmp/PocketMine-MP/blob/stable/src/world/World.php#L1909
https://github.com/pmmp/PocketMine-MP/blob/stable/src/entity/Entity.php#L735